.github/workflows/CICD.yml

```
@@ -73,33 +73,10 @@ jobs:
         run: PHP_CS_FIXER_IGNORE_ENV=1 vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.php --verbose --diff --dry-run
 
   check_js:
-    name: 2️⃣ JS Node ${{ matrix.node-version }} - Code Style errors & Compilation
-    runs-on: ubuntu-latest
+    uses: ./.github/workflows/js_check.yml
```

This interface is being annoying again.

For consistency this should be the same as the php one. For ease of reading I'd suggest name first. I have no strong opinion about `uses`/`needs`.
```suggestion
    name: 2️⃣ JS front-end
    uses: ./.github/workflows/js_check.yml
```